vite.config.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+
+// eslint-disable-next-line import/no-unresolved
+import VueI18nPlugin from '@intlify/unplugin-vue-i18n/vite';
+import vue from '@vitejs/plugin-vue';
+import { defineConfig } from 'vite';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'url';
+
+// import.meta.env isn't available in this file.
+const buildMode = process.env.NODE_ENV;
+// The default is es2020, but we need es2022 or later because Web Forms uses
+// top-level await.
+const buildTarget = 'es2022';
+
+// Requests to forward to nginx
+const proxyPaths = [
+  '/v1',
+  '/-',
+  '/enketo-passthrough',
+  '/client-config.json',
+  '/version.txt'
+];
+const devServer = {
+  port: 8989,
+  proxy: Object.fromEntries(proxyPaths.map(path => [path, 'http://localhost:8686'])),
+  // Because we proxy to nginx, which itself proxies to Backend and other
+  // things, the dev server doesn't need to allow CORS. CORS is already limited
+  // by default, but we just don't need it at all.
+  cors: false
+};
+
+export default defineConfig({
+  plugins: [
+    vue(),
+    VueI18nPlugin({
+      include: resolve(dirname(fileURLToPath(import.meta.url)), './src/locales/**'),
+      compositionOnly: false,
```

Comment:
Maybe in some point in future we can only use composition APIs and remove this line

Reply:
I feel like we've been slowly migrating our components from the Options API to the Composition API, which I think is a good thing. Even if we continue to use the Options API for some components, those components can still use `setup()` I think. We'll have to remove this option once we move to v12 of Vue I18n: see getodk/central#1027.